packages/react-storage/src/components/StorageBrowser/context/elements/definitions.ts

```diff
@@ -33,15 +34,21 @@ type IconVariant =
   | 'file'
   | 'folder'
   | 'menu'
-  | 'paginate-next'
-  | 'paginate-previous'
+  | Exclude<PaginateVariant, 'paginate-current'>
   | 'refresh'
   | 'search'
   | 'sort-ascending'
   | 'sort-descending'
   | 'sort-indeterminate'
   | 'vertical-kebab';
 
+export type IconElementProps = React.ComponentProps<typeof IconElement>;
+
+export type ButtonElementProps = React.ComponentProps<typeof ButtonElement>;
+type ButtonElementVariant = 'action-submit' | 'search-submit' | PaginateVariant;
+
+export type ListItemVariant = PaginateVariant;
```

Comment:
just for my understanding, what should be the difference between these two types when using them? Like is `PaginateVariant` only for the paginate control whereas `ListItemVariant` is for any list item, or something like that?